README.md

```diff
@@ -152,3 +152,289 @@ commit it in this repo.
 ```
 just tf grafana apply
 ```
+## Updating the Deployment
+
+This deployment uses
+[cardano-parts](https://github.com/input-output-hk/cardano-parts), which
+periodically releases new features and `cardano-node` versions. Therefore,
+it’s necessary to update the `cardano-parts` version in this repository. This
+section provides a step-by-step guide to perform the update.
+
+### Environment Setup
+
+Before starting, ensure that you have the appropriate environment:
+
+- **Using the `dev-deployer` machine**: The `direnv` tool will automatically
+  load the Nix development shell.
+- **Using a local copy**: If working from a local repository, run the
+  following command to enable the necessary Nix features:
+
+  ```bash
+  nix --extra-experimental-features fetch-closure develop
```

Comment:
For those working regularly in this repo, it may be worth mentioning that the `fetch-closure` experimental feature can be added to their nix.conf file so it doesn't need to be included manually on the cli each time.  This would then also allow using direnv to automatically load the nix develop default shell when entering the local repo directory.